src/test/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/PaymentRequestMapperTest.java

```diff
@@ -0,0 +1,88 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.client.payhub.request.PaymentRequest;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.config.SiteConfiguration;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.exception.SiteNotFoundException;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.exceptions.InvalidMessageException;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.model.PaymentMessage;
+
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.bulkscan.payment.processor.data.producer.SamplePaymentMessageData.DCN_1;
+import static uk.gov.hmcts.reform.bulkscan.payment.processor.data.producer.SamplePaymentMessageData.DCN_2;
+import static uk.gov.hmcts.reform.bulkscan.payment.processor.data.producer.SamplePaymentMessageData.PO_BOX;
+import static uk.gov.hmcts.reform.bulkscan.payment.processor.data.producer.SamplePaymentMessageData.paymentMessage;
+
+@ExtendWith(MockitoExtension.class)
+public class PaymentRequestMapperTest {
+
+    @Mock
+    private SiteConfiguration siteConfig;
+
+    private PaymentRequestMapper paymentRequestMapper;
+
+    @BeforeEach
+    void setUp() {
+        paymentRequestMapper = new PaymentRequestMapper(siteConfig);
+    }
+
+    @Test
+    public void should_return_valid_PaymentRequest_when_paymentMessage_is_valid() {
+        // given
+        when(siteConfig.getSiteIdByPoBox(PO_BOX)).thenReturn("A123");
+        PaymentRequest expectedPaymentRequest = new PaymentRequest(
+            "case_number_3333",
+            ImmutableList.of(DCN_1, DCN_2),
+            false,
+            "A123"
+        );
+
+        // when
+        PaymentRequest paymentRequest = paymentRequestMapper.mapPaymentMessage(paymentMessage(
```

Comment:
Perhaps not for this PR, but in other projects we've made a mistake of having this "sample data" class prepare data and then we've got lots of tests expect specific details that this class returns. I think it would be best if tests fully defined their own data. As mentioned, we don't have to do in this PR.